Cabal-tests/tests/ParserTests.hs

```diff
@@ -250,24 +271,38 @@ formatRoundTripTest fp = testCase "roundtrip" $ do
     x <- parse contents
     let contents' = showGenericPackageDescription x
     y <- parse (toUTF8BS contents')
-    -- previously we mangled licenses a bit
-    let y' = y
+
+    let checkField field =
+          unless (field x == field y) $
```

Comment:
While at it, could you possible use [(@?)](https://hackage-content.haskell.org/package/tasty-hunit-0.10.2/docs/Test-Tasty-HUnit.html#v:-64--63-) or `assertBool` instead of raw `assertFailure`?